core/src/main/scala/better/files/TeeOutputStream.scala

```diff
@@ -8,11 +8,11 @@ import java.io.OutputStream
   * @param outs
   */
 class TeeOutputStream(outs: OutputStream*) extends OutputStream {
-  override def write(b: Int)                             = tryAll(outs)(_.write(b))
-  override def flush()                                   = tryAll(outs)(_.flush())
-  override def write(b: Array[Byte])                     = tryAll(outs)(_.write(b))
-  override def write(b: Array[Byte], off: Int, len: Int) = tryAll(outs)(_.write(b, off, len))
-  override def close()                                   = tryAll(outs)(_.close())
+  override def write(b: Int)                                    = tryAll(outs)(_.write(b))
+  override def flush()                                          = tryAll(outs)(_.flush())
+  override def write(b: Array[Byte] | Null)                     = tryAll(outs)(_.write(b))
+  override def write(b: Array[Byte] | Null, off: Int, len: Int) = tryAll(outs)(_.write(b, off, len))
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.